interface/requests/mod.go

```diff
@@ -37,3 +35,7 @@ const (
 	OpPsaVerifyMessage     OpCode = 0x0019
 	OpListKeys             OpCode = 0x001A
```

Comment:
Maybe we need a warning notice here, or a test of some sort, to also modify the `IsValid` method is an opcode bigger than the biggest one is added.
Same for all the `IsValid` methods 😬